src/python/test/test_dxclient.py

```diff
@@ -6218,6 +6218,56 @@ def test_build_workflow_with_bill_to(self):
         new_gwf = json.loads(run("dx build --globalworkflow --bill-to {} --json {}".format(org_id, workflow_dir)))
         self.assertEqual(new_gwf["billTo"], org_id)
 
+    @unittest.skipUnless(testutil.TEST_ISOLATED_ENV,
+                         'skipping test that would create global workflows')
+    def test_dx_add_list_remove_users_of_workflows(self):
```

Comment:
Minor: may want to say globalworkflows here to be more precise

Reply:
(y)